conda_build/api.py

```diff
@@ -281,3 +282,13 @@ def update_index(dir_paths, config=None, force=False, check_md5=False, remove=Fa
 
     for path in dir_paths:
         update_index(path, config, force=force, check_md5=check_md5, remove=remove)
+
+
+def cut_project(name, path, conf=None):
+    cookie = Cookie(name, path, conf=conf)
+    create_project_skeleton(cookie)
+    cookie.initial_commit()
+    python_ver = input("Enter python version [2.7]: ") or '2.7'
```

Comment:
Please never, ever require interactive input. Make it a CLI argument and pass it as a keyword arg here.